src/ui/public/index_patterns/_index_pattern.js

```diff
@@ -38,14 +38,14 @@ export default function IndexPatternFactory(Private, Notifier, config, kbnIndex,
   });
 
   const mapping = mappingSetup.expandShorthand({
-    title: 'string',
-    timeFieldName: 'string',
+    title: 'keyword',
```

Comment:
I ended up changing this to `text` anyway. I explained the reason in f654def5998d20e1155a4a259cf0067b097a52c5.